amplipi/streams.py

```diff
@@ -789,10 +789,28 @@ def send_cmd(self, cmd):
     """
     try:
       if cmd in self.supported_cmds:
+        if cmd == "love":
+          src_config_folder = f'{utils.get_folder("config")}/srcs/v{self.vsrc}'
+          with open(f'{src_config_folder}/.config/pianobar/currentSong', 'r+', encoding='utf-8') as file:
+            data = file.read()
+            chunks = data.split(",,,")
+            if len(chunks) >= 5:
+              liked_state = chunks[4].strip()
+              if liked_state == "0":
+                chunks[4] = "1"
+              elif liked_state == "1":
+                chunks[4] = "0"
+            modified_data = ",,,".join(chunks)
+            file.seek(0)
+            file.truncate()
+            file.write(modified_data)
+            file.flush()
```

Comment:
Inside the command handling function, intercept the love command and rewrite the metadata file with the opposite liked state
This is gross but also the only realistic way to have the liked status be visible, so oh well ig